tools/container-converter/src/main.rs

```diff
@@ -30,7 +30,10 @@ async fn main() -> Result<(), String> {
 
     match container_converter::run(request).await {
         Ok(response) => {
-            println!("{:?}", response);
+            let response_serialized = serde_json::to_string(&response)
+                .map_err(|err| format!("Failed serializing conversion request. {:?}", err))?;
+
+            println!("Successful nitro conversion: {:?}", response_serialized);
```

Comment:
The string corresponds to a search prefix defined here: https://github.com/fortanix/app-test-infra/blob/5abda4112fb64559eb4e0064cf78dc7e4a29cb0a/python/test_app.py#L1383

Reply:
Oh I'm surprised it was working without the prefix strings matching.

Reply:
It wasn't, the old version of app-test-infra didn't have the code mentioned.